README.md

```diff
@@ -44,13 +44,16 @@ It's necessary to connect:
 - [mount-observe](https://snapcraft.io/docs/mount-observe-interface) interface;
 - [system-trace](https://snapcraft.io/docs/system-trace-interface) interface;
 - `sys-kernel-debug-sched-features` plug into the [system-files](https://snapcraft.io/docs/system-files-interface) interface;
+- `posix-mq` plug into the `posix-mq-provider` slot through the [posix-mq](https://snapcraft.io/docs/posix-mq-interface) interface;
+
 
 
 ```bash
 sudo snap connect rt-tests:process-control
 sudo snap connect rt-tests:mount-observe
 sudo snap connect rt-tests:system-trace
 sudo snap connect rt-tests:sys-kernel-debug-sched-features
+sudo snap connect rt-tests:posix-mq rt-tests:posix-mq-provider
 ```
 > **_NOTE:_** The `sys-kernel-debug-sched-features` interface gets auto-connected when installed from the store.
```

Comment:
```suggestion
> [!NOTE]
> - The `sys-kernel-debug-sched-features` interface gets auto-connected when installed from the store.
> - The `pmqtest` app is granted access to use POSIX message queues via the [posix-mq](https://snapcraft.io/docs/posix-mq-interface)  interface.
```
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts